ruby3.2-ffi.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: ruby3.2-ffi
+  version: 1.17.0
+  epoch: 0
+  description: Ruby FFI
+  copyright:
+    - license: BSD-3-Clause And MIT
+  dependencies:
+    runtime:
+      - autoconf
+      - automake
+      - libffi-dev
+      - libtool
+      - ruby-3.2
+      - ruby3.2-bundler
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - libffi-dev
+      - libtool
+      - ruby-3.2
+      - ruby-3.2-dev
+      - ruby3.2-bundler
+
+vars:
+  gem: ffi
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: ecfb225096ae76ba2a5e8115f046bd0ac23095e6
+      repository: https://github.com/ffi/ffi.git
+      tag: v${{package.version}}
+
+  - runs: |
+      git submodule update --init --recursive
+      bundle install
+      rake compile -- --enable-system-libffi
+      gem install ffi
```

Comment:
This will install the gem from the repository and not the one built locally. You'd pass something like `./ffi-${{package.version}}`

Reply:
Turns out the gemspec is fetching version dynamically. So passing `version-tag` returns a `Not found` error.

[sdk] ❯ gem install ffi
Fetching ffi-1.17.0-aarch64-linux-gnu.gem
Successfully installed ffi-1.17.0-aarch64-linux-gnu
Ignoring debug-1.7.1 because its extensions are not built. Try: gem pristine debug --version 1.7.1
Parsing documentation for ffi-1.17.0-aarch64-linux-gnu
Installing ri documentation for ffi-1.17.0-aarch64-linux-gnu
Done installing documentation for ffi after 0 seconds
1 gem installed

Reply:
I think we need to pass `--local` here. From the log output, it appears to be pulling it down and not using the one we've built

Reply:
That worked.

[sdk] ❯ gem install ffi -- --local
Successfully installed ffi-1.17.0-aarch64-linux-gnu
Ignoring debug-1.7.1 because its extensions are not built. Try: gem pristine debug --version 1.7.1
Parsing documentation for ffi-1.17.0-aarch64-linux-gnu
Done installing documentation for ffi after 0 seconds
1 gem installed